tests/Doctrine/Tests/DBAL/Query/Expression/CompositeExpressionTest.php

```diff
@@ -22,6 +22,13 @@ public function testCount()
         $this->assertEquals(2, count($expr));
     }
 
+    public function testToString()
+    {
+        $expr = new CompositeExpression(CompositeExpression::TYPE_AND, array('u.username = "foo"', 'u.name = "bar"'));
+
+        static::assertEquals('(u.username = "foo") AND (u.name = "bar")', (string) $expr);
```

Comment:
Yeah but why not `$this` like in every other test, too? Even though it might work, it still looks weird and makes you think something special is going on here... Or am I missing something?